graph/pull.go

```diff
@@ -428,10 +428,10 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 	}
 
 	if verified {
-		out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified"))
-	} else {
-		out.Write(sf.FormatStatus(tag, "Pulling from %s", repoInfo.CanonicalName))
+		log.Printf("Image manifest verified for %s:%s", repoInfo.CanonicalName, tag)
 	}
+	out.Write(sf.FormatStatus(tag, "Pulling from %s", repoInfo.CanonicalName))
```

Comment:
Shouldn't this happen *before* the `"image manifest verified"` log? I just find the sequence a bit weird:

```
Image manifest verified for foo/bar:baz
Pulling from foo/bar
```